### PR TITLE
Add first blocks to the Education Edition

### PR DESCRIPTION
**Pull request by Kaooot**

### CODE REVIEW

**Comment by lukeeey** [comment]
Wouldnt it be better to put these in another directory to seperate them from normal blocks? Maybe name it "education".

**Comment by Kaooot** [comment]
Idk

**Comment by Janmm14** [comment]
I think it should be separated somehow.

The server needs to make sure if edumode is disabled that those blocks cannot be used.

There should probably be a method like isEdumodeExclusive() (maybe shorter somehow)

**Comment by lukeeey** [comment]
```java
GoMint.isEduMode()
```
Then that matches up with the method in the game itself (`MinecraftGame::isEduMode`)

**Comment by Janmm14** [comment]
I meant Block and Item should have such a method

File: gomint-server/src/main/java/io/gomint/server/network/packet/PacketStartGame.java [hunk 1]
```diff
@@ -121,6 +122,7 @@ public void serialize( PacketBuffer buffer, int protocolID ) {
         buffer.writeBoolean( this.unknown1 );
         buffer.writeLLong( this.currentTick );
         buffer.writeSignedVarInt( this.enchantmentSeed );
+        buffer.writeBoolean( this.educationEdition );
```
Comment (geNAZt) [inline gomint-server/src/main/java/io/gomint/server/network/packet/PacketStartGame.java]: This is not part of the protocol in 1.5.2 or 1.6.0.8 where does this come from?

**Review by geNAZt — CHANGES_REQUESTED** [review]

File: gomint-server/src/main/java/io/gomint/server/network/packet/PacketStartGame.java (hunk 1 above)
Comment (lukeeey) [reply gomint-server/src/main/java/io/gomint/server/network/packet/PacketStartGame.java]: @geNAZt He put it in the wrong place but https://github.com/pmmp/PocketMine-MP/blob/master/src/pocketmine/network/mcpe/protocol/StartGamePacket.php#L200

Comment (geNAZt) [reply gomint-server/src/main/java/io/gomint/server/network/packet/PacketStartGame.java]: Please remove the boolean, i will merge it then

Comment (lukeeey) [reply gomint-server/src/main/java/io/gomint/server/network/packet/PacketStartGame.java]: @geNAZt You do realise you have access to push commits to their fork if they enabled it on the PR?

**Comment by lukeeey** [comment]
Why did the build fail with some random bs about NormalGenerator?

**Pull request closed without merging**